client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -114,7 +120,7 @@ const ListingCardImageError =
 
 const testPrice = function priceTest(card, mountedCard) {
   it('Should display formatted price', () => {
-    expect(mountedCard.text()).to.include(formatMoney(card.props.listing.price, card.props.listing.priceUnit));
+    expect(mountedCard.text()).to.include(formatMoney(card.props.listing.price.get(':money'), card.props.listing.price.get(':priceUnit')));
```

Comment:
I think these tests would be better if we'd match against a literal formatted value, in this case `'199,00 €'` (?)